be/src/olap/tablet.cpp

```diff
@@ -3318,6 +3333,13 @@ Status Tablet::update_delete_bitmap(const RowsetSharedPtr& rowset,
               << ", cur max_version: " << cur_version << ", transaction_id: " << txn_id
               << ", cost: " << watch.get_elapse_time_us() << "(us), total rows: " << total_rows;
 
+    if (do_correctness_check) {
```

Comment:
we can just use `rowset->num_rows() > 0` here, don't need to pass do_correctness_check param.